aws/amazon-gamelift-plugin-unity
# Feature/anywhere Sample game

ASG4

Updated the Sample Game to use SDK5 and work with GameLift Anywhere

To test this you can follow these steps:
- run `./Scripts~/windows/release.ps1`
- create a new Unity project
- enable [unity nuget](https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-engines-unity-using.html#integration-engines-unity-prerequisites)
- [install the plugin generated from first step as tarball](https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-engines-unity-using.html#integration-engines-unity-SDK-plugin)
- create following file in root of new unity project:
  ```
  // GameLiftConfiguration.yaml
  version: 1
  WebsocketUrl: 
  FleetId: 
  FleetLocation: 
  ComputeName: 
  CurrentProfileName: 
  CurrentRegion: 
  ```
  Fill out all the fields with correct values. The plugin will generate this file in other PRs
- In the new unity project, GameLift > Import Sample Game
- Choose either GameLift > Apply server settings or GameLift > Apply client settings.
- Hit play


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
